OptimizelySDK.sln

```diff
@@ -16,10 +16,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 	ProjectSection(SolutionItems) = preProject
 		CHANGELOG.MD = CHANGELOG.MD
 		CONTRIBUTING.md = CONTRIBUTING.md
-		OptimizelySDK.Package\OptimizelySDK.nuspec = OptimizelySDK.Package\OptimizelySDK.nuspec
-		OptimizelySDK.Package\pack.ps1 = OptimizelySDK.Package\pack.ps1
 		README.md = README.md
-		OptimizelySDK.Package\verifysn.ps1 = OptimizelySDK.Package\verifysn.ps1
```

Comment:
Moved template to repo root and consuming code in [Actions workflow](https://github.com/optimizely/csharp-sdk/blob/master/.github/workflows/csharp_release.yml#L168)